### PR TITLE
Fix #3536: Merged home_activity.xml files

### LINKED ISSUE #3536: Merge home_activity.xml into single xml file
Currently there are 3 versions of `home_activity.xml` file merge it into single xml file.

We can use https://text-compare.com/ to compare two versions of this file and for all the differences we can create variables in `dimens.xml` file and use it accordingly.

**Note**: In PR, make sure you add before and after screenshot of mobile-portrait, mobile-landscape, tablet-portrait and tablet-landscape for comparison and make sure that there is not difference between before and after UI.

### PR DESCRIPTION
**Pull request by mukulasija**
##Merged 3 home_activity xml files into single xml file 
   Fix #3536 
   Just deleted 2 of the 3 home_activity files because after comparing in text compare I found that not a single character was different in all three files and also added the layout_height parameter in dimen file.

Following are the before and after screenshots:

||||
|:----------------------------------------:|:-----------------------------------------:|:-----------------------------------------: |
| tab_old_portrait
| tab_new_portrait
| tab_old_landscape
| tab_new_landscape
| phone_old_landscape
| phone_new_landscape
| phone_old_portrait
| phone_new_portrait

## Essential Checklist

- [x] The PR title and explanation each start with "Fix #bugnum: " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] Any changes to [scripts/assets](https://github.com/oppia/oppia-android/tree/develop/scripts/assets) files have their rationale included in the PR explanation.
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary code changes from Android Studio ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#undo-unnecessary-changes)).
- [x] The PR is made from a branch that's **not** called "develop" and is up-to-date with "develop".
- [x] The PR is **assigned** to the appropriate reviewers ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section)).

### CODE REVIEW

**Comment by mukulasija** [comment]
@rt4910 PTAL

**Comment by BenHenning** [comment]
Hi. As of today, some main reviewers have taken time off for the next few weeks, so it may take a little while before we can look at this PR. We appreciate your patience while some of our team members recharge. We'll be fully returning on 4 January 2021.

**Review by Arjupta — APPROVED** [review]
@mukulasija nice the chabges looks good to me.

**Comment by Arjupta** [comment]
@mukulasija do resolve conflicts and update your PR so that it can be merged soon.

File: app/src/main/res/layout/home_activity.xml [hunk 1]
```diff
@@ -18,7 +18,7 @@
     <FrameLayout
       android:id="@+id/home_fragment_placeholder"
       android:layout_width="match_parent"
-      android:layout_height="0dp"
+      android:layout_height="@dimen/home_activity_frame_layout_height"
```
Comment (rt4914) [inline app/src/main/res/layout/home_activity.xml]: You can directly hard code this here as `0dp` and remove the dimens value that you have created from `dimens.xml` file.

**Review by rt4914 — CHANGES_REQUESTED** [review]
@mukulasija Thanks. One nit change suggested.

Also please update the title of this PR to:
`Fix #3536: Merged home_activity.xml files`

File: app/src/main/res/values/dimens.xml [hunk 2]
```diff
@@ -587,7 +587,7 @@
   <dimen name="section_title_text_view_layout_margin_start">28dp</dimen>
   <dimen name="section_title_text_view_layout_margin_top">28dp</dimen>
   <dimen name="section_title_text_view_layout_margin_end">32dp</dimen>
-
+  
   <!-- AudioLanguageFragment -->
```
Comment (Arjupta) [inline app/src/main/res/values/dimens.xml]: (Nit change) @mukulasija this extra space can also be removed.

File: app/src/main/res/layout/home_activity.xml (hunk 1 above)
Comment (mukulasija) [reply app/src/main/res/layout/home_activity.xml]: @rt4914 I am done with the requested changes... PTAL

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

File: app/src/main/res/values/dimens.xml (hunk 2 above)
Comment (rt4914) [reply app/src/main/res/values/dimens.xml]: Resolving as this change has been done

**Pull request merged**